### ISSUE TITLE
[Category] Short description

### ISSUE THREAD

**Issue by ToeMonster** [sp614x/optifine#5803]
## Steps to Reproduce
1. What do you do...
2. ...to find this bug?
I basically go through the nether portal, also going back to the overworld doesn't fix it, it makes it same

## OptiFine Version

OptiFine HD U G9 pre20

## Installation Method
 I used standalone installer

## Other Installed Mods
 Nothing

## F3 Debug Screenshot
 https://imgur.com/a/xb5QsaZ

## Additional Information

Here is a video of the bug -< https://www.reddit.com/r/Minecraft/comments/nz503v/any_idea_how_to_fix_this_happens_everytime_i_go/

**Comment by Scorpion34** [comment]
First things first use Preview 21, not 20 this could fix it but if not what's likely happening is Optifine at its current state does not like big worlds so anything exceeding maybe 3 or 5 gigabytes it'll cause insane amounts of lag and chunk loading issues especially upgrading older version worlds to the new 1.17 I experienced this issue with my hardcore world that I lost due to these types of chunk issues so if preview 21 doesn't fix it my best recommendation would be to wait for newer versions of Optifine until then you're kinda stuck.

**Comment by KaiAF** [comment]
Could not have said it better, I really think if you are playing hardcore, either dont update to 1.17 or dont use optifine until all the major world glitches are fixed.

Sorry your hardcore world was lost @Scorpion34

**Comment by KaiAF** [comment]
All though this bug should have been fixed in pre 21. Correct me if I'm wrong

**Comment by Scorpion34** [comment]
> Could not have said it better, I really think if you are playing hardcore, either dont update to 1.17 or dont use optifine until all the major world glitches are fixed.
> 
> Sorry your hardcore world was lost @Scorpion34

Yeah its no big deal encouraged me to hop on here and address the issues so other people won't make the same mistakes for 1.17 though you're really not missing anything if you go back a version I mean we got axolotls, deepslate and goats... nothing too extraordinary so I'll probably continue playing 1.16.5 whilst testing 1.17 for bugs or glitches which I'm almost certain 1.18 will be train wreck for optifine so I'm hoping SP is gonna work ahead schedule for 1.18, anyways appreciate the support but yeah optifine for huge worlds is really struggling.
